test/show.jl

```diff
@@ -164,6 +164,137 @@ end
                                2 columns omitted"""
 end
 
+@testset "GroupedDataFrame displaysize test" begin
```

Comment:
can you please add tests for `GroupedDataFrame` with 0 and with 1 group?

Reply:
@bkamins  Ok, will do. Quick side question: the commit with which I merged (`d6277e40d`, message `Synchronize ByRow and Tables.subset with Tables.jl 1.8 (#3158)`) doesn’t precompile (throws `ByRow not defined`). Not sure how to “un-sync” my fork — should I wait for you to commit a fix and then resync my fork?

Reply:
Have you made sure to update Tables.jl to 1.8.1?
What is the problem exactly on your end?

All works OK on my machine:
```
(@v1.8) pkg> activate .
  Activating new project at `~/WORK`

(WORK) pkg> add DataFrames#main
    Updating git-repo `https://github.com/JuliaData/DataFrames.jl.git`
    Updating registry at `~/.julia/registries/General`
    Updating git-repo `https://github.com/JuliaRegistries/General.git`
   Resolving package versions...
    Updating `~/WORK/Project.toml`
  [a93c6f00] + DataFrames v1.4.0 `https://github.com/JuliaData/DataFrames.jl.git#main`
    Updating `~/WORK/Manifest.toml`
  [34da2185] + Compat v4.2.0
  [a8cc5b0e] + Crayons v4.1.1
  [9a962f9c] + DataAPI v1.10.0
  [a93c6f00] + DataFrames v1.4.0 `https://github.com/JuliaData/DataFrames.jl.git#main`
  [864edb3b] + DataStructures v0.18.13
  [e2d170a0] + DataValueInterfaces v1.0.0
  [59287772] + Formatting v0.4.2
  [41ab1584] + InvertedIndices v1.1.0
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [e1d29d7a] + Missings v1.0.2
  [bac558e1] + OrderedCollections v1.4.1
  [2dfb63ee] + PooledArrays v1.4.2
⌃ [08abe8d2] + PrettyTables v1.3.1
  [189a3867] + Reexport v1.2.2
  [a2af1166] + SortingAlgorithms v1.0.1
  [3783bdb8] + TableTraits v1.0.1
  [bd369af6] + Tables v1.8.1
  [56f22d72] + Artifacts
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [9fa8497b] + Future
  [b77e0a4c] + InteractiveUtils
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA v0.7.0
  [9e88b42a] + Serialization
  [6462fe0b] + Sockets
  [2f01184e] + SparseArrays
  [10745b16] + Statistics
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode
  [e66e0078] + CompilerSupportLibraries_jll v0.5.2+0
  [4536629a] + OpenBLAS_jll v0.3.20+0
  [8e850b90] + libblastrampoline_jll v5.1.1+0
        Info Packages marked with ⌃ have new versions available
Precompiling project...
  1 dependency successfully precompiled in 18 seconds. 19 already precompiled.

julia> using DataFrames

julia>
```

Reply:
That might have been it. I re-cloned my fork and re-`Pkd.dev`ed it — working now!